### DISCUSSION TITLE
Displaying script on the workstation screen

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#1991] [General]
I created a PowerShell script for maintenance on Windows. It uses an existing script from Tactical for disk cleanup, combining it with other commands for general system maintenance, applying updates, checking disks, etc.
I added the script to a task on the workstations and set it to "manual," so it only runs when I initiate it.
My question is: Is it possible to have the script appear on the PC screen when it is being executed?

**Comment by silversword411** [comment]
No, TRMM agent runs as SYSTEM and everything is in background.

**Comment by bbrendon** [comment]
you could save the output to a log file an have a program pop up that watches it like http://www.baremetalsoft.com/baretailpro/index.php

You might have buffer issues though. GLHF :)
